### PR TITLE
Use WithProductionDoorOverlay on fake weap

### PR DESCRIPTION
**Pull request by abcdefg30**
At current bleed trying to place a fake weap crashes the game.

### CODE REVIEW

**Comment by penev92** [comment]
:+1:

File: mods/ra/rules/fakes.yaml [hunk 1]
```diff
@@ -39,8 +39,8 @@ WEAF:
 	Bib:
 	RenderBuilding:
 		Image: WEAP
-	WithIdleOverlay@ROOF:
-		Sequence: idle-top
+	WithProductionDoorOverlay:
```
Comment (pchote) [inline mods/ra/rules/fakes.yaml]: This probably wants to stay as `WithIdleOverlay`.

Comment (abcdefg30) [inline mods/ra/rules/fakes.yaml]: > 18:23 (abcdefg30) hm, pchote somehow https://github.com/OpenRA/OpenRA/pull/8045#discussion_r29160908 doesn't work out, as the fake plays the open sequence in a loop now 
18:23 (+pchote) ah, right

**Comment by obrakmann** [comment]
not only bleed crahes, but the release, too :/

**Comment by obrakmann** [comment]
:+1:

**Comment by obrakmann** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-%28bleed%29/_compare/4dcca57c57eb057eaa3c54bc410c39405266e598%5E...4dcca57c57eb057eaa3c54bc410c39405266e598)

File: mods/ra/rules/fakes.yaml (hunk 1 above)
Comment (Mailaender) [inline mods/ra/rules/fakes.yaml]: You could invent a new sequence `idle-door` with just one frame of the closed door.

**Comment by Mailaender** [comment]
Sorry, as that is my fault. I wonder why it crashed though.

File: mods/ra/rules/fakes.yaml (hunk 1 above)
Comment (Mailaender) [inline mods/ra/rules/fakes.yaml]: Or just revert partly https://github.com/Mailaender/OpenRA/commit/8f93d7b5d76a20174349830016a6ada05cb27ee4 to bring back `idle-top`.

**Comment by Mailaender** [comment]
Now I get it. Should have tested the fake weapons factory after @pchote's changes again. https://github.com/OpenRA/OpenRA/pull/7709 would have detected it by the way to draw some attention to that boring pull request which will help a lot maintaining this repository.

**Pull request merged**